core/options.go

```diff
@@ -26,6 +27,7 @@ func ParseOptions() (Options, error) {
 	options := Options{
 		Threads:           flag.Int("threads", 0, "Number of concurrent threads (default number of logical CPUs)"),
 		OutDir:            flag.String("out", ".", "Directory to write files to"),
+		OutFile:           flag.String("outf", "aquatone_urls.txt", "File to dump alive hosts to"),
```

Comment:
I think this feature is useful enough to just write out the file by default like it does with `aquatone_report.html`, so we can remove this flag IMO.